docs/data/toolpad/core/components/sign-in-page/SlotPropsSignIn.js

```diff
@@ -16,17 +16,19 @@ export default function SlotPropsSignIn() {
             `Signing in with "${provider.name}" and credentials: ${formData.get('email')}, ${formData.get('password')} and checkbox value: ${formData.get('tandc')}`,
           )
         }
+        slots={{ checkbox: RememberCheckbox }}
         slotProps={{
+          form: { noValidate: true },
           emailField: { variant: 'standard', autoFocus: false },
           passwordField: { variant: 'standard' },
           submitButton: { variant: 'outlined' },
-          rememberMe: {
+          checkbox: {
```

Comment:
`checkbox` is more generic than `rememberMe` so I think it's an improvement, but could we make it more generic even as any type of content could go in this slot, such as a small disclaimer for example, or multiple checkboxes?
Not sure if something like `formFooter` would be a good name...
Anyway these are just some thoughts!

Reply:
Also I see we have a `forgotPasswordLink` slot too, so if when implementing these you saw that there's a benefit to these slots being more specific then it should be fine.

Reply:
Moving those to "plug-in" components seems good to me if they're complex enough!